test/wallaby/dsl/actions/file_test.exs

```diff
@@ -0,0 +1,41 @@
+defmodule Wallaby.Actions.FileTest do
+  use Wallaby.SessionCase, async: true
+
+  setup %{server: server, session: session} do
+    page =
+      session
+      |> visit(server.base_url <> "forms.html")
+
+    {:ok, %{page: page}}
+  end
+
+  describe "attaching a file to a form" do
+    test "by name", %{page: page} do
+      page
+      |> attach_file("file", path: "test/support/fixtures/file.txt")
+
+      assert find(page, "#file_field") |> has_value?("C:\\fakepath\\file.txt")
+    end
+
+    test "by DOM ID", %{page: page} do
+      page
+      |> attach_file("file_field", path: "test/support/fixtures/file.txt")
+
+      assert find(page, "#file_field") |> has_value?("C:\\fakepath\\file.txt")
+    end
+
+    test "by label", %{page: page} do
+      page
+      |> attach_file("File", path: "test/support/fixtures/file.txt")
+
+      assert find(page, "#file_field") |> has_value?("C:\\fakepath\\file.txt")
+    end
+  end
+
+  test "attaching a non-extant file does nothing", %{page: page} do
```

Comment:
This test defines what happens when a bad path is given. Not sure how much use that test is but I figured it would be useful to define the behaviour.